src/test/java/subway/LineAcceptanceTest.java

```diff
@@ -0,0 +1,228 @@
+package subway;
+
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.jdbc.Sql;
+import subway.line.LineRequest;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static io.restassured.RestAssured.given;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("지하철 노선 관련 기능")
+@Sql({"/sql/station-data.sql"})
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+public class LineAcceptanceTest {
+
+    /**
+     * Given: 새로운 지하철 노선 정보를 입력하고,
+     * When: 관리자가 노선을 생성하면,
+     * Then: 해당 노선이 생성되고 노선 목록에 포함된다.
+    */
+    @DisplayName("지하철 노선을 생성한다.")
+    @Test
+    void createLine() {
+
+        // given
+        LineRequest lineRequest = new LineRequest();
+        lineRequest.setName("신분당선");
+        lineRequest.setColor("red");
+        lineRequest.setUpStationId(1);
+        lineRequest.setDownStationId(2);
+        lineRequest.setDistance(10);
+
+
+        // when
+        // then
+        assertThat(createLine(lineRequest).statusCode()).isEqualTo(HttpStatus.CREATED.value());
+
+    }
+
+    /**
+     * Given: 여러 개의 지하철 노선이 등록되어 있고,
+     * When: 관리자가 지하철 노선 목록을 조회하면,
+     * Then: 모든 지하철 노선 목록이 반환된다.
+     */
+    @DisplayName("지하철 노선 목록을 조회한다.")
+    @Test
+    void showLines() {
+        // given
+        LineRequest lineRequestRed = new LineRequest();
+        lineRequestRed.setName("신분당선");
+        lineRequestRed.setColor("bg-red-600");
+        lineRequestRed.setUpStationId(1);
+        lineRequestRed.setDownStationId(2);
+        lineRequestRed.setDistance(10);
+
+        assertThat(createLine(lineRequestRed).statusCode()).isEqualTo(HttpStatus.CREATED.value());
```

Comment:
```suggestion
        assertThat(createLine("신분당선", "bg-red-600", 1, 2, 10).statusCode()).isEqualTo(HttpStatus.CREATED.value());
```

이런형태는 어떨까요? 😄 

LineRequest 를 생성하는 과정이 행을 많이 차지하고 있어서 실제 집중할 테스트에 집중이 안되는 것 같아요 😢

Reply:
참고해서 수정했습니다!